lib/internal/webidl.js

```diff
@@ -0,0 +1,161 @@
+'use strict';
+
+const {
+  MathAbs,
+  MathMax,
+  MathMin,
+  MathPow,
+  MathTrunc,
+  NumberIsNaN,
+  NumberMAX_SAFE_INTEGER,
+  NumberMIN_SAFE_INTEGER,
+} = primordials;
+
+const {
+  codes: {
+    ERR_INVALID_ARG_VALUE,
+  },
+} = require('internal/errors');
+const { kEmptyObject } = require('internal/util');
+
+const integerPart = MathTrunc;
```

Comment:
Is there a specific reason for this variable declaration? I might have missed it.

Reply:
Added a link to the webidl operation definition.